### PR TITLE
modules rename 'avro' => 'schemedkafka'

### PR DESCRIPTION
**Pull request by gadomsky**

### CODE REVIEW

File: benchmarks/src/test/scala/pl/touk/nussknacker/engine/benchmarks/serialization/avro/AvroBenchmark.scala [hunk 1]
```diff
@@ -7,10 +7,10 @@ import org.apache.flink.api.common.ExecutionConfig
 import org.apache.flink.api.common.typeinfo.TypeInformation
 import org.apache.flink.formats.avro.typeutils.{LogicalTypesGenericRecordAvroTypeInfo, LogicalTypesGenericRecordWithSchemaIdAvroTypeInfo}
 import org.openjdk.jmh.annotations._
-import pl.touk.nussknacker.engine.avro.kryo.AvroSerializersRegistrar
-import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.ConfluentUtils
-import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.client.{CachedConfluentSchemaRegistryClientFactory, MockSchemaRegistryClient}
-import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.kryo.SchemaIdBasedAvroGenericRecordSerializer
+import pl.touk.nussknacker.engine.schemedkafka.kryo.AvroSerializersRegistrar
```
Comment (dswiecki) [inline benchmarks/src/test/scala/pl/touk/nussknacker/engine/benchmarks/serialization/avro/AvroBenchmark.scala:10]: in avro related classes I would prefer pl.touk.nussknacker.engine.schemedkafka.avro package, in unreleated just pl.touk.nussknacker.engine.schemedkafka

Comment (gadomsky) [reply benchmarks/src/test/scala/pl/touk/nussknacker/engine/benchmarks/serialization/avro/AvroBenchmark.scala:10]: I see the point, but on the other hand we have other dimensions
* serialization vs deserialization
* sink vs source
* encode vs decode
etc.

Comment (gadomsky) [reply benchmarks/src/test/scala/pl/touk/nussknacker/engine/benchmarks/serialization/avro/AvroBenchmark.scala:10]: Another possiblity is to have:
* deserializatin/json
* deserializatin/avro
* deserializatin/common

But I am not sure if those "subdomains" are big enough that we have to devide them into avro/json/common related sub-sub-domains.

Comment (gadomsky) [reply benchmarks/src/test/scala/pl/touk/nussknacker/engine/benchmarks/serialization/avro/AvroBenchmark.scala:10]: I'm resolving this issue, but only for some time. Lets tackle that later.

**Review by dswiecki — APPROVED** [review]

**Pull request merged**